.github/workflows/ci.yml

```
@@ -697,6 +697,16 @@ jobs:
       - run: python -m pytest modin/tests/pandas/dataframe/test_reduce.py
       - run: python -m pytest modin/tests/pandas/dataframe/test_udf.py
       - run: python -m pytest modin/tests/pandas/dataframe/test_window.py
+      - run: python -m pytest modin/tests/pandas/interop/test_binary.py
```

```suggestion
      - run: python -m pytest modin/tests/native_df_mode/pandas/interop/test_binary.py
```
I envision we could implement a native df mode for polars so let's change the test directory stucture a bit.

---

Done